app/services/gias_csv_importer.rb

```diff
@@ -87,54 +88,68 @@ def initialize(csv_path)
 
   def call
     invalid_records = []
-    records = []
+    school_records = []
+    trust_records = []
+    trust_associations = Hash.new { |h, k| h[k] = [] }
```

Comment:
This is clever! 🤓 I've never seen this before.

Reply:
I've only learnt it myself recently, I think it's an improvement over adding a guard clause to create the empty array should it be missing.